docs/source/markdown/podman-machine-init.1.md.in

```diff
@@ -160,6 +160,11 @@ Example: `-v "$HOME/git:$HOME/git:ro,security_model=none"`
 Default volume mounts are defined in *containers.conf*.  Unless changed, the default values
 is `$HOME:$HOME`.
 
+**Note on Windows Subsystem for Linux (WSL)**
+Since the entire `C:` drive is mounted at `/mnt/c` by default in WSL, passing
```

Comment:
AFAIK, this is not just restricted to the C drive, is it? Doesn't windows mount all drives that way, e.g. /mnt/d and so on? Maybe add this as well in the note here

Reply:
Good point, it does seem that that's the expected behavior. I'll make the update.